fetch2/src/main/java/com/tonyodev/fetch2/HttpUrlConnectionDownloader.kt

```diff
@@ -5,14 +5,15 @@ import java.io.InputStream
 import java.net.CookieHandler
 import java.net.HttpURLConnection
 import java.net.URL
-import java.util.Collections
+import java.util.*
```

Comment:
```suggestion
import java.util.Collections
```

Reply:
yes, formatting is not necessary.
but I think we should apply the newest official stable kotlin code-style for code which modified.